lib/glific_web/schema/organization_types.ex

```diff
@@ -0,0 +1,85 @@
+defmodule GlificWeb.Schema.OrganizationTypes do
+  @moduledoc """
+  GraphQL Representation of Glific's Organization DataType
+  """
+
+  use Absinthe.Schema.Notation
+  import Absinthe.Resolution.Helpers, only: [dataloader: 1]
+
+  alias Glific.Repo
+  alias GlificWeb.Resolvers
+
+  object :organization_result do
+    field :organization, :organization
+    field :errors, list_of(:input_error)
+  end
+
+  object :organization do
+    field :id, :id
+    field :name, :string
+    field :bsp_key, :string
+    field :contact_name, :string
+    field :email, :string
+    field :wa_number, :string
+
+    field :bsp, :bsp do
+      resolve(dataloader(Repo))
+    end
+  end
+
+  @desc "Filtering options for organizations"
+  input_object :organization_filter do
+    @desc "Match the name"
+    field :name, :string
+
+    @desc "Match the email"
+    field :email, :string
+
+    field :contact_name, :string
```

Comment:
Do you think we should put @desc for these fields as well or remove from the above two?

Reply:
Yes, not needed